uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request.controller.js

```diff
@@ -57,43 +47,52 @@ define([
     function calculateToilExpiryDate () {
       // blocks the expiry date from updating if this is an existing request
       // and user is not a manager or admin
-      if (!vm.canManage && vm.request.id) {
-        return $q.resolve(vm.request.toil_expiry_date);
+      if (!parentCtrl.canManage && parentCtrl.request.id) {
+        return $q.resolve(parentCtrl.request.toil_expiry_date);
       }
 
       // skips calculation of expiration date if request never expires
       // according to admin setting
-      if (!vm.requestCanExpire) {
-        vm.request.toil_expiry_date = false;
+      if (!parentCtrl.requestCanExpire) {
+        parentCtrl.request.toil_expiry_date = false;
         return $q.resolve(false);
       }
 
       return getReferenceDate().catch(function (errors) {
-        if (errors.length) vm.errors = errors;
+        if (errors.length) parentCtrl.errors = errors;
         return $q.reject(errors);
```

Comment:
this errors are being thrown to be used where?

Reply:
@AkA84 These errors are then shown as a "red block above `submit` button block".